### DISCUSSION TITLE
New release this year?

### DISCUSSION THREAD

**Discussion by C-Duv** [jacobsalmela/tccutil#73] [General]
Is a new release planned this year?

I am using Homebrew to install `tccutil` ([`brew install tccutil`](https://formulae.brew.sh/formula/tccutil)) which installs the 1.4.0 version.

But this version lacks some of the fixes I need (for #61 and #70).

I could use `brew install --head tccutil` to get a more recent code, but it feels wrong/too unstable.

I have workarounds but if some release is in the pipe I rather know before continuing in this path :)

Thanks

**Comment by jacobsalmela** [comment]
Yeah, it needs a brew-pr-bump.  I'll look at that soon.

**Reply by C-Duv** [reply]
Just out of curiosity: you have Brew-specific things to do for a release?

**Reply by jacobsalmela** [reply]
Someone beat me to it: https://github.com/Homebrew/homebrew-core/pull/192534
